cmd/current.go

```diff
@@ -0,0 +1,33 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/tfversion/tfversion/pkg/current"
+	"github.com/tfversion/tfversion/pkg/helpers"
+)
+
+const (
+	currentExample = "# Print the current active version of Terraform\n" +
+		"tfversion current"
+)
+
+var (
+	currentCmd = &cobra.Command{
+		Use:     "current",
+		Short:   "Print the current active version of Terraform",
+		Example: currentExample,
+		Run: func(cmd *cobra.Command, args []string) {
+			err := current.CheckCurrentVersion()
```

Comment:
We usually don't return an error from the function we call from the command, but just let that function exit if an error occurs.

Reply:
Resolved in fd905807e0e559b2d4bb6743350e353e17ebc392